Bottom: b71cbca149b235e357916cf1e74eb60e498125cc
Top:    4baa9f3e2cdcdfd30c2096881f95445230ff882b
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 19:20:05 +0000

find.c: Fix flag passed to ListStart() in DRCFind().

We should have set the flag before calling ListStart(), as it marks
the first item with that flag.


---

diff --git a/src/find.c b/src/find.c
index 9198df4..68b65d7 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3360,8 +3360,8 @@ DRCFind (int What, void *ptr1, void *ptr2, void *ptr3)
           ListStart (What, ptr1, ptr2, ptr3, flag);
           DoIt (flag, true, true);
           DumpList ();
-          ListStart (What, ptr1, ptr2, ptr3, flag);
           flag = FOUNDFLAG;
+          ListStart (What, ptr1, ptr2, ptr3, flag);
           Bloat = 0;
           drc = true;
           DoIt (flag, true, true);
